src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +28,13 @@
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<Object> machines, Machine value);
 
     /**
      * Call the method doWork() from each machine.
      * This method should be able to accept a list of bulldozers as well as list of trucks.
      *
      * @param machines - the list of machines
      */
-    void startWorking(List<Object> machines);
+    void startWorking(List<? extends Machine> machines);
```

Comment:
The 'startWorking' method is correctly using a bounded wildcard, but consider if it could be better to use the generic type 'T' as the parameter to ensure that the method works with the specific machine type that the service is intended for.